feature_tests/dart/lib/src/lib.g.dart

```diff
@@ -656,9 +656,15 @@ final class _Writeable {
   _Writeable() : _ffi = _diplomat_buffer_writeable_create(0);
   
   String finalize() {
-    final string = Utf8Decoder().convert(_diplomat_buffer_writeable_get_bytes(_ffi).asTypedList(_diplomat_buffer_writeable_len(_ffi)));
-    _diplomat_buffer_writeable_destroy(_ffi);
-    return string;
+    try {
+      final buf = _diplomat_buffer_writeable_get_bytes(_ffi);
+      if (buf == ffi.Pointer.fromAddress(0)) {
```

Comment:
I assume this works. `==` looks risky due to object comparison vs value comparison, but I don't know how this type works in Dart.

Reply:
This compares addresses: https://api.dart.dev/stable/3.4.1/dart-ffi/Pointer/operator_equals.html